Snakefile

```diff
@@ -239,6 +240,8 @@ rule build_shapes:
         offshore_shapes="resources/" + RDIR + "shapes/offshore_shapes.geojson",
         africa_shape="resources/" + RDIR + "shapes/africa_shape.geojson",
         gadm_shapes="resources/" + RDIR + "shapes/gadm_shapes.geojson",
+        subregion_shapes=("resources/" + RDIR + "shapes/subregion_shapes.geojson"
```

Comment:
I'm not a great fan of this else cases; what about we specify "resources/" + RDIR + "shapes/subregion_shapes.geojson" and the file is empty when no subregions are used?